defaults/stroopwafel_interface_default.py

```diff
@@ -0,0 +1,286 @@
+        radius_2 = utils.get_zams_radius(mass_2[index], metallicity_2[index])
+        roche_lobe_tracker_1 = radius_1 / (location.dimensions[a] * (1 - eccentricity[index]) * utils.calculate_roche_lobe_radius(mass_1[index], mass_2[index]))
+        roche_lobe_tracker_2 = radius_2 / (location.dimensions[a] * (1 - eccentricity[index]) * utils.calculate_roche_lobe_radius(mass_2[index], mass_1[index]))
+        location.properties['is_rejected'] = 0
+        if (mass_2[index] < constants.MINIMUM_SECONDARY_MASS) or (location.dimensions[a] <= (radius_1 + radius_2)) \
+        or roche_lobe_tracker_1 > 1 or roche_lobe_tracker_2 > 1:
+            location.properties['is_rejected'] = 1
+            num_rejected += 1
+    return num_rejected
+
+
+if __name__ == '__main__':
+
+    # STEP 1 : Import and assign input parameters for stroopwafel 
+    parser=argparse.ArgumentParser()
+    parser.add_argument('--num_systems', help = 'Total number of systems', type = int, default = num_systems)  
+    parser.add_argument('--num_cores', help = 'Number of cores to run in parallel', type = int, default = num_cores)
+    parser.add_argument('--num_per_core', help = 'Number of systems to generate in one core', type = int, default = num_per_core)
+    parser.add_argument('--debug', help = 'If debug of COMPAS is to be printed', type = bool, default = debug)
+    parser.add_argument('--mc_only', help = 'If run in MC simulation mode only', type = bool, default = mc_only)
+    parser.add_argument('--run_on_hpc', help = 'If we are running on a (slurm-based) HPC', type = bool, default = run_on_hpc)
+    parser.add_argument('--output_filename', help = 'Output filename', default = output_filename)
+    parser.add_argument('--output_folder', help = 'Output folder name', default = output_folder)
+    namespace, extra_params = parser.parse_known_args()
+
+    start_time = time.time()
+    #Define the parameters to the constructor of stroopwafel
+    TOTAL_NUM_SYSTEMS = namespace.num_systems #total number of systems you want in the end
+    NUM_CPU_CORES = namespace.num_cores #Number of cpu cores you want to run in parellel
+    NUM_SYSTEMS_PER_RUN = namespace.num_per_core #Number of systems generated by each of run on each cpu core
+    debug = namespace.debug #If True, will print the logs given by the external program (like COMPAS)
+    run_on_hpc = namespace.run_on_hpc #If True, it will run on a clustered system helios, rather than your pc
+    mc_only = namespace.mc_only # If you dont want to do the refinement phase and just do random mc exploration
+    output_filename = namespace.output_filename #The name of the output file
+    output_folder = os.path.join(os.getcwd(), namespace.output_folder)
+
+    # Set commandOptions defaults - these are Compas option arguments
+    commandOptions = dict()
+    commandOptions.update({'--outputPath' : output_folder}) 
+    commandOptions.update({'--logfile-delimiter' : 'COMMA'}) 
```

Comment:
This should not be hardcoded here! What if the user tries to change it from pythonSubmit, its not being respected.

Reply:
Ya I agree there's a bit of a degeneracy here.

Ultimately, it's always going to be a little ambiguous which file wins out in the scenario where sw and pySub are both used. But in this case, since people would be actually running the sw_interface file, I figured it makes more sense to accept the sw value in stead of the pySub value. 

I'm open to discussion / disagreement on this, but in the meantime I'll try to make it clearer in the comments.